server/cluster/cluster.go

```diff
@@ -62,6 +62,7 @@ const (
 	// since the once the store is add or remove, we shouldn't return an error even if the store limit is failed to persist.
 	persistLimitRetryTimes = 5
 	persistLimitWaitTime   = 100 * time.Millisecond
+	retry                  = 3
```

Comment:
This name is arbitrary, I prefer a more specific one to explain its usage clearly.